backend/src/common/services/calculation-progress/calculation-progress.service.ts

```diff
@@ -20,15 +21,20 @@ class CalculationProgressService {
     private achievementService: AchievementService;
     private goalService: GoalService;
     private userAchievementsService: UserAchievementService;
+    private notificationService: NotificationService;
 
+    /* eslint-disable max-params */
     public constructor(
+        /* eslint-enable max-params */
```

Comment:
why do we need to disable linter?

Reply:
and enable right after one line))

Reply:
@stpalamar linter does not allows me to pass more than 3 params to constructor, but I need to add Notification Service to it and 3 others are needed for calculation which Serhii implemented.
If you know how to make it without linter disabling, I would be very thankful))

Reply:
you can pass an object to constructor

Reply:
Here an example:
https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/pull/387/files#diff-33831ca28b49ebcc1f5b505ce580b4d87cdc3caf7cb4dcbfebee441cf6497aaeR14